### ISSUE TITLE
Support for JIT compilation

### ISSUE THREAD

**Issue by viktorstrate** [richardgill/tailwind-override#18]
Thank you for the project, it makes working with React and Tailwind much easier.

I do however encounter a problem when using Tailwinds experimental [JIT compilation](https://tailwindcss.com/docs/just-in-time-mode) feature.

When running the following:

```js
var {overrideTailwindClasses} = require("tailwind-override")
console.log(overrideTailwindClasses('bg-white bg-[#222]'))
```

I get back `bg-white bg-[#222]`, but I expected to get back only `bg-[#222]`

**Comment by richardgill** [comment]
@viktorstrate I think your logic is correct and this would work - I'm just thinking about how to rewrite this package not to depend on 16MB of json, so I'll try and fix both together.

**Comment by viktorstrate** [comment]
I understand, thank you for your efforts. This project takes react + tailwind to the next level by enabling composable styling.

**Comment by richardgill** [comment]
Hi @viktorstrate,

I've just released 0.5.0 on npm which supports tailwind jit 🥳. (Here are some relevant test cases: https://github.com/richardgill/tailwind-override/blob/master/test/testCases.ts#L37-L50)

Let me know if you run into any problems let me know.

**Comment by viktorstrate** [comment]
That's great to hear 🎉

I did however find something that I would use it for that it didn't handle correctly, and that's css functions.
I often use the `calc()` function to mix css units, but it didn't seem to pick up on them in `v0.5.0`.

```js
const overrideTailwindClasses = require("tailwind-override").overrideTailwindClasses
overrideTailwindClasses('w-[calc(10px + 25%)] w-[200px] w-24')
// output: "w-[calc(10px + 25%)] w-24" 
```
https://runkit.com/embed/21xk12610cmq
